App.js

```diff
@@ -1,31 +1,22 @@
 require("dotenv").config()
 const bodyParser = require("body-parser");
 const express = require("express");
-const mongoose = require("mongoose");
+
+const runDb = require("./src/db.js");
+
 const swaggerjsdocs = require("swagger-jsdoc");
 const swaggerUi = require("swagger-ui-express")
-const jwt = require("jsonwebtoken")
-const bcrypt = require("bcrypt")
-
-const {DoctorRoute} = require("./Routes/doctor.js")
-const {PatientRoute} = require("./Routes/patient.js")
 
-const Doctors = require("./models/doctorModel");
-const Patients = require("./models/patientModel")
-const Hospital = require("./models/hospitalModel")
+const {doctorRoute} = require("./src/Routes/doctor.routes.js")
+const {patientRoute} = require("./src/Routes/patient.routes.js")
+const { authRoute } = require("./src/Routes/authentication.routes.js");
+const { generalRoute } = require("./src/Routes/general.routes.js");
 
-// const port = process.env.PORT
+const port = process.env.PORT
 
 
```

Comment:
Removed the unneccesary requires as App.js file now doesnt have any direct routes